webrtc-stats.html

```diff
@@ -1256,6 +1258,39 @@
                 </p>
               </dd>
               <dt>
+                <dfn><code>estimatedPlayoutTimestamp</code></dfn> of type <span class=
+                "idlMemberType"><a>DOMHighResTimeStamp</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Only valid for remote sources. This is the estimated playout time of this track.
+                  The playout time is the NTP timestamp of the last played out audio sample or video
+                  frame that has a known timestamp, extrapolated with the time elapsed since it was
+                  played out. This is the "current time" of the track and can be present even if
+                  there is no audio or video currently playing.
```

Comment:
Do we want to say that is is expected to be able use estimatedPlayoutTimestamp from two tracks of the same source to calculate out of sync issues?